BankLibrary/Account.cs

```diff
@@ -3,22 +3,30 @@
 namespace BankLibrary
 {
     public delegate void AccountCreated(string message);
-    
+    public delegate void AccountClosed(string message);
+    public delegate void MoneyPutted(string message);
+    public delegate void MoneyWithdrawn(string message);
```

Comment:
it will be 
```c#
public event Action <string> Created;
public event Action<string> Closed;
public event Action<string> Putted;
public event Action<string> Withdrawn;
```